packages/block-library/src/paragraph/style.scss

```diff
@@ -29,7 +29,7 @@
 }
 
 p.has-background {
-	padding: $block-bg-padding--v $block-bg-padding--h;
+	padding: 1.25em 2.375em;
```

Comment:
I think it's odd that you're changing `$block-bg-padding--v` and `$block-bg-padding--h` to use `em` values, and then _not_ using them here. Is that intentional?

Reply:
Ah you're right, I just changed the vars _after_ I had pushed the changes in this file so it got lost.
Fixed in [`ae64ed0`](https://github.com/WordPress/gutenberg/pull/24523/commits/ae64ed0b02ab8899d520a41cf8239c02cec5a4c7)